resources/src/topicHeader.cpp

```diff
@@ -49,6 +49,7 @@ const char* CONTROL_LOADPARAM="/kraken/control/loadparam_srv";
 
 //sensor topics
 const char* SENSOR_IMU="/kraken/sensor/imu";
+const char* SENSOR_IMU_NEW="/kraken/sensor/imu_new";
```

Comment:
@athityakumar I have no clue. I ran `rosmake` from inside the `sparton` package. You are not facing this error?